homeassistant/components/enphase_envoy/diagnostics.py

```diff
@@ -40,6 +42,16 @@ async def async_get_config_entry_diagnostics(
         {
             "entry": entry.as_dict(),
             "envoy_firmware": coordinator.envoy.firmware,
+            "supported_features": [
+                feature.name
+                for feature in SupportedFeatures
+                if feature in coordinator.envoy.supported_features
+            ],
+            "phase_mode": coordinator.envoy.phase_mode,
+            "phase_count": coordinator.envoy.phase_count,
+            "active_phasecount": coordinator.envoy.active_phase_count,
+            "ct_count": coordinator.envoy.ct_meter_count,
+            "ct_consumption_meter": coordinator.envoy.consumption_meter_type,
             "data": coordinator.data,
```

Comment:
Can we add this in a separate PR?

Reply:
I guess so, but was trying to make sure all would be avaialble for same functionality when released

Reply:
Removed from this PR, will post as seperate PR